QUARKUS-874.md

```diff
@@ -0,0 +1,36 @@
+# QUARKUS-874 Production support for Windows
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-874
+
+The goal is to ensure some level of support and identify existing limitations on Quarkus side for Windows.
+
+## Scope of the testing
+The selected Test Suites will be executed on the selected Windows Platforms to ensure functionality is fully accomplished.
+
+### Selected Windows Platforms
+- Windows Server 2019
+
+### Selected Test Suites
+ - Start-Stop Test Suite
+   - In Github CI - for Pull Requests
+   - New Jenkins Jobs
+ - Beefy Test Suite
+   - In Github CI - for Pull Requests and Daily runs
+   - New Jenkins Jobs
+ - QuickStarts Test Suite
+   - New Jenkins Jobs
+ - Combinations of Quarkus extensions Test Suite
+   - In Github CI - for Pull Requests and Daily runs
+ - Workshop Test Suite
+   - New Jenkins Jobs
+
+### Platform testing
+Tests are going to be executed against all tested versions of databases, RHEL and OpenShift versions. 
+Both the JVM and native mode is going to be tested.
+
+## Impact on resources
+- New Windows instances in our Jenkins environments
+  - Windows instance must support Native compilation (GraalVM, VS Build Tools) and Docker (Windows Containers)
+
```

Comment:
Should we state the existing limitation around testcontainers / Linux containers on Windows? I think this will be eventually addressed, so we don't need to state it here.

Reply:
I would mention that, it causes us a lot of troubles. And the available solution doesn't work out of the box / we (Tomas mainly) spent a lot of hours on this topic.

Reply:
Added as part of the Impact on resources section and I provided also additional context.